src/bundle.ts

```diff
@@ -61,213 +60,40 @@ export class Bundler extends Transform {
 
   _flush(done: (error?: any) => void) {
     this._buildBundles().then((bundles: Map<string, string>) => {
-      for (const fragment of this.config.allFragments) {
-        const file = this.analyzer.getFile(fragment);
-        console.assert(file != null);
-        const contents = bundles.get(fragment);
+      for (const filename of bundles.keys()) {
+        const filepath = pathFromUrl(this.config.root, filename);
+        let file =
+            this.analyzer.getFile(filepath) || new File({path: filepath});
```

Comment:
if the analyzer has never seen the fragment before, that's bad. Consider just not handling that case or throwing a special error here so that this isn't swallowed.